types/DIVAProtocolPolygon.md

```diff
@@ -73,19 +87,18 @@ keccak256(abi.encode("DIVAProtocolPolygon",abi.encode(1)))
 
 ### Encoding/Decoding
 
-A value of 99.9 would be submitted on-chain using the following bytes:
-
-`0x0000000000000000000000000000000000000000000000056a6418b505860000`
+For example, if the reference asset is ETH/USD with a price of $2,819.35, and the collateral token is DAI/USD at a price of $0.9996, then the hexadecimal of the bytes-encoded expected response for this query-- ready to be submitted by a reporter --is:
 
+`0x00000000000000000000000000000008490b0f122862a11da8c6ac40000000000000000000000000000000000000000000c08415c61401a79c97e10c00000000`
 
 ## Dispute Considerations
 
 Reporters should use care in selecting data sources and choosing the algorithm to combine them.
 
-- The prediction market nature of DIVA Protocol means that the query could have a result that is a price, a temperature, a boolean, a sports score, or any different format for the data.  
-- The result should be the correct one AT THE TIME of expiration. This means that if the prediction market bets on the price of BTC/USD on Feb 1st at midnight, the query should always return the price of BTC/USD at midnight on Feb 1st, even if it get called/requested again. 
+- The prediction market nature of DIVA Protocol means that the query could have a result that is a price, a temperature, a boolean, a sports score, or any different format for the data.
```

Comment:
We have an agreement with Diva that we're only reporting for ETH/USD and BTC/USD for now, and we'll add more possible data points in the future. Perhaps we should mention that here?